web/lib/api/cache/stats.ts

```diff
@@ -95,7 +97,7 @@ FROM cache_hits
 WHERE (
   (${builtFilter.filter})
 )
-GROUP BY response.body ->> 'model'::text;
+GROUP BY response.body ->> 'model'::text, request.provider;
```

Comment:
model should probably change too

Reply:
yeah - it's fine. it's for an old endpoint on graphQL no one really uses... This will all be rewritten when we move to REST